File: howManySundays.cpp
int howManySundays(int n, string startDay) {
    using namespace std;
    vector<string> days = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
    int a = distance(days.begin(), find(days.begin(), days.end(), startDay));
    int b = a + n;
    int a1 = a / 7;
    int b1 = b / 7;
    return b1 - a1;
}

File: howManySundays.py
def howManySundays(n, startDay):

    week = ['Sunday', 'Monday', 'Tuesday', 'Wednesday',
            'Thursday', 'Friday', 'Saturday']
    startIndex = -1

    for i in range(len(week)):
        if week[i] == startDay:
            startIndex = i
            break

    return math.floor((n + startIndex) / len(week))
